### PR TITLE
Docs: Configuring Cluster Operator - Cluster Operator Modularization

### PR DESCRIPTION
**Pull request by dmuntima**
### Type of change

- Documentation

### Description

Continue modularizing Cluster Operator

### CODE REVIEW

**Comment by dmuntima** [comment]
@ncbaratta, I have created a PR for section 3.4. Operator configuration (does not include 3.4.1) of the Cluster Operator chapter.

The existing content has to be rewritten to a procedure. I have moved the existing content to a proc module, updated ID, added assembly-module text in the comment, and made minor modifications. Please review. Thanks!

File: documentation/book/configuring-cluster-operator.adoc [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+include::common/attributes.adoc[]
+
+//This module is to replace section 3.4. Operator configuration.
+
+[id='operator-configuration-{context}']
+= Configuring Cluster Operator
+
+You can configure the Cluster Operator using the following environment variables.
+
+.Prerequisites
```
Comment (scholzj) [inline documentation/book/configuring-cluster-operator.adoc]: Should we delete the prerequisites title if there aren't any?

Comment (ncbaratta) [reply documentation/book/configuring-cluster-operator.adoc]: Yes!

**Comment by dmuntima** [comment]
Closing this PR as Jakub has created https://github.com/strimzi/strimzi-kafka-operator/pull/742/ based on the new list of assemblies and modules.

**Pull request closed without merging**